client/components/forms/form-text-input/test/index.jsx

```diff
@@ -1,62 +1,65 @@
-import { expect } from 'chai';
-import { shallow } from 'enzyme';
-import { spy } from 'sinon';
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
 import FormTextInput from '../';
 
 describe( '<FormTextInput />', () => {
 	test( 'should add the provided class names', () => {
```

Comment:
I generally find such tests close to worthless TBH. To me, it's a good example of testing an implementation detail, something that's discouraged for testing and not that valuable to test in general.

Reply:
Hm. If the component changes depending on props, wouldn't we want to test how it behaves with different props? Those classes are only applied if certain props are passed to the component. I tend to agree regarding the `className` though.

If we want to be less specific we could use a snapshot test to make sure the component doesn't change unintendedly.